src/main/java/controller/BlackJackGame.java

```diff
@@ -0,0 +1,170 @@
+
+import domain.card.Card;
+import domain.card.CardShuffler;
+import domain.card.Deck;
+import domain.card.RandomCardShuffler;
+import domain.result.PlayerProfitStatistics;
+import domain.user.*;
+import view.InputView;
+import view.OutputView;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static java.util.stream.Collectors.joining;
+import static java.util.stream.Collectors.toList;
+import static view.OutputView.*;
+
+public class BlackJackGame {
+    private static final int FIRST = 0;
+
+    public static void main(String[] args) {
+        Players players = inputPlayers();
+        Dealer dealer = new Dealer();
+        initializeCardsOfAllUsers(players, dealer, new RandomCardShuffler());
+        showCardsOfAllUsers(players, dealer);
+        askWantMoreCardToAllPlayers(players);
+        distributeCardToDealer(dealer);
+        showDealerAndPlayersResult(players, dealer);
+        showDealerAndPlayersProfit(players, dealer);
+    }
+
+    private static Players inputPlayers() {
+        List<PlayerName> playerNames = inputPlayerNames();
+        List<Player> players = playerNames.stream()
+                .map(BlackJackGame::convertNamesAndBettingMoneyIntoPlayers)
+                .collect(toList());
+        return new Players(players);
+    }
+
+    private static List<PlayerName> inputPlayerNames() {
+        OutputView.printPleaseInputNames();
+        return convertInputNamesIntoPlayerNames(InputView.inputNames());
+    }
+
+    private static List<PlayerName> convertInputNamesIntoPlayerNames(List<String> playerNames) {
+        return playerNames.stream()
+                .map(String::trim)
+                .map(PlayerName::new)
+                .collect(toList());
+    }
+
+    private static Player convertNamesAndBettingMoneyIntoPlayers(PlayerName playerName) {
+        OutputView.printAskHowMuchBettingToPlayer(playerName);
+        BettingMoney bettingMoney = convertAmountIntoBettingMoney(InputView.inputAmount());
+        return new Player(playerName, bettingMoney);
+    }
+
+    private static BettingMoney convertAmountIntoBettingMoney(BigDecimal amount) {
+        return new BettingMoney(amount);
+    }
+
+    private static void initializeCardsOfAllUsers(Players players, Dealer dealer, CardShuffler cardShuffler) {
+        Deck.initializeDeck(cardShuffler);
+        Deck.distributeCardsToPlayersAndDealer(players, dealer);
+        OutputView.printHandedOutTwoCardsToPlayers(players);
+    }
+
+    private static void showCardsOfAllUsers(Players players, Dealer dealer) {
+        showCardsOfDealer(dealer);
+        showCardsOfPlayers(players);
+    }
+
+    private static void showCardsOfDealer(Dealer dealer) {
+        Card firstCard = dealer.getCards().get(FIRST);
+        String firstCardTypeName = firstCard.getType().getTypeName();
+        String firstCardSymbolName = firstCard.getSymbol().getSymbolName();
+        OutputView.printCardsHeldByDealer(firstCardTypeName, firstCardSymbolName);
```

Comment:
```suggestion
        OutputView.printCardsHeldByDealer(firstCard);
```
위와 같이 Card를 파라미터로 전달해서, ``OutputView.printCardsHeldByDealer`` 내부에서 symbol과 type을 꺼내서 출력하면 되지 않을까요 🤔

Reply:
과도하게 원시값을 `View`에 전달하려고 했던 것 같습니다. 피드백 감사합니다 :)